nixos/instantiate.sh

```diff
@@ -5,4 +5,4 @@ set -euo pipefail
 
 FLAKE="${1}"
 
-nix path-info --json --derivation "${FLAKE}.config.system.build.toplevel" | jq '{ "path": .[0].path }'
+nix path-info --derivation "${FLAKE}.config.system.build.toplevel" | jq -R '{ "path": . }'
```

Comment:
Very minor suggestion, but I try to use long option names for commands that are run non-interactively (to make them more self-documenting):

```suggestion
nix path-info --derivation "${FLAKE}.config.system.build.toplevel" | jq --raw-input '{ "path": . }'
```

Reply:
Fixed